src/components/elements/input/CheckboxGroupInput.tsx

```diff
@@ -57,6 +67,7 @@ const CheckboxGroupInput: React.FC<CheckboxGroupInputProps> = ({
             sx={{
               color: disabled ? 'text.disabled' : 'text.primary',
               '&.Mui-focused': { color: 'text.primary' },
+              ...labelSx,
```

Comment:
I added `labelSx` here, instead of just using `maxWidth` like everywhere else, in order for the label to actually be smaller than the width of the whole component, since these are horizontal radio buttons.